packages/common/src/types/TextEditor.ts

```diff
@@ -196,6 +196,12 @@ export interface EditableTextEditor extends TextEditor {
    */
   revealDefinition(range?: Range): Promise<void>;
 
+  /**
+   * Reveal definition aside
+   * @param range A {@link Range range}
+   */
+  revealDefinitionAside(range?: Range): Promise<void>;
```

Comment:
I would be tempted to just make this an option passed to `revealDefinition`. Slightly mixed feelings about it being on the editor, as it opens a new editor, but probably fine. Cc/ @AndreasArvidsson

Reply:
I'm fine with either way. You are revealing a definition of something that exists in a particular editor so I think it's fine that it's on the editor type.

Reply:
Makes sense. WDYT about options object on existing function instead of new function. See also below re adding it to followLink instead of revealDefinition

Reply:
As a said I'm fine with either adding an argument to existing function or adding a new function. With regards to the editor declaring capabilities it might be better with a separate function?